src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -142,7 +142,7 @@ def __call__(
                 plain tuple.
 
         Returns:
```

Comment:
@mishig25 not really sure why this is not displayed. It's 1-to-1 the same as in Transformers